propel/schema.xml

```diff
@@ -1,51 +1,53 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<database name="pizzaService">
-    <table name="ingredients" phpName="Ingredient">
+<database name="pizzaService" defaultIdMethod="native" namespace="Pizzaservice\Propel\Models">
+    <table name="ingredient" phpName="Ingredient">
```

Comment:
Unfortunatly you did not understand what I meant correctly. The table-names should always be plural because they contain multiple rows/objects of the same type. So a table named ingredients contains many objects of type ingredient.
However when referencing an ingredient from another table (for example the pizza_ingredient table) you only reference a single ingredient. So you refernce the id of a single row, so the row should be named ingredient_id. (=the ID of one ingredient)